src/common/Centerer.js

```diff
@@ -1,50 +0,0 @@
-/* @flow strict-local */
-import React, { PureComponent } from 'react';
-import { View } from 'react-native';
-import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
-
-import type { Node as React$Node } from 'react';
-import styles, { createStyleSheet } from '../styles';
-
-const componentStyles = createStyleSheet({
-  centerer: {
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'space-around',
-    alignItems: 'center',
-  },
-  inner: {
-    width: '100%',
-    maxWidth: 480,
```

Comment:
This `maxWidth` looks like it was probably doing something. In particular, on `AccountPickScreen`, if your screen is wide (like on a tablet, or in landscape on a phone) then I think this is what stops the items in the accounts list from stretching across the whole width.

Maybe move this down into the two callsites? Those are better places for this layout choice to appear anyway.

Reply:
Makes sense.

Hmm, here's a question about `Screen`'s interface:

When you pass `centerContent` to a `Screen`, that gives `justifyContent: 'center'` to a container element (so, centers that element's children vertically), but doesn't set `alignItems: 'center'`, to center content horizontally; `alignItems` is left with its default, 'stretch'. This means children with a `maxWidth` aren't centered horizontally, but rather sit to the left. Children without a `maxWidth` are stretched to fill the container (so I guess they are effectively centered).

Would it be an improvement to `Screen`'s interface to have `centerContent` set both `justifyContent` and `alignItems` to 'center'?

Reply:
Yeah, good find -- I think that would be an improvement.